File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Punkapi SDK exists test

require_once __DIR__ . '/../punkapi_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = PunkapiSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# Punkapi SDK exists test

require "minitest/autorun"
require_relative "../Punkapi_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = PunkapiSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
